### DISCUSSION TITLE
I still can't see the image preview using @alias

### DISCUSSION THREAD

**Discussion by studyforhuihui** [geriyoco/vscode-image-gallery#130] [Q&A]
Where am I not configured
![image](https://github.com/user-attachments/assets/a5b0f753-c909-4e09-a34b-ea2b51263fe6)
![image](https://github.com/user-attachments/assets/4c6543d1-fd63-4d38-bddd-2fe8c15fb05a)

**Comment by fanurs** [comment]
Hi. I'm not sure if `gutterpreview` is from our extension.

However, I do recall another popular extension called [Image Preview](https://marketplace.visualstudio.com/items?itemName=kisstkondoros.vscode-gutter-preview), and it has `gutterpreview` in its settings? See: https://github.com/search?q=repo%3Akisstkondoros%2Fgutter-preview+gutterpreview&type=code

**Reply by studyforhuihui** [reply]
Sorry, I replied to the wrong place
